common/lib/api-client/models/person.js

```diff
@@ -23,6 +23,10 @@ module.exports = {
       jsonApi: 'hasMany',
       type: 'profiles',
     },
+    category: {
```

Comment:
I'd also say, even if we don't want to rename the model. JSON:API allows the ability to define relationships with different names. So maybe at a minimum this relationship should be `prison_category` (or `prisoner_category` is that's the correct term).

Reply:
> Will see what can be done